ci/playbooks/pre-doc.yml

```diff
@@ -43,7 +43,7 @@
 
         - name: Download and extract golang
           ansible.builtin.unarchive:
-            src: "https://golang.org/dl/go{{ go_version | default('1.20.14') }}.linux-amd64.tar.gz"
```

Comment:
I noticed the job failure and trying to fix this. Should we change the default or use the variable `go_version` in https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/doc.yaml#L6